stable/atlantis/templates/statefulset.yaml

```diff
@@ -72,6 +72,10 @@ spec:
               exec:
                 command: ["/bin/sh", "-c", "cp /etc/secret-gitconfig/gitconfig /home/atlantis/.gitconfig && chown atlantis /home/atlantis/.gitconfig"]
           {{- end}}
+          command:
```

Comment:
Should only be set if `.Values.command` is set, otherwise you override the `ENTRYPOINT` with nothing don't you?

Reply:
fix it.